docs/_posts/2018-01-23-grep-on-the-fly-in-spacevim.md

```diff
@@ -0,0 +1,87 @@
+---
+title: "Grep on the fly in SpaceVim"
+categories: feature
+excerpt: "Run grep Asynchronously and on the fly as you type in SpaceVim, quick searching whole project or opened files."
+image: https://user-images.githubusercontent.com/13142418/35278709-7856ed62-0010-11e8-8b1e-e6cc6374b0dc.gif
+---
+
+# Asynchronous grep on the fly
+
+FlyGrep means **grep on the fly**, it will update the result as you type. Of course, it is running
+asynchronously. Before using this feature, you need to install a searching tool. FlyGrep can be
+interfaced with different searching tools like: `ag`, `rg`, `ack`, `pt` and `grep`, select any one
```

Comment:
You may want to say “FlyGrep supports other search tools, like: `ag`, `rg`, `ack`, `pt` and `grep`. Choose one you like.”

Reply:
`other search tools` means we have build tools, but no, we flygrep is just a framework for searching, it need a command line tool, and communicate with the command via std.

Reply:
Ohh, "FlyGrep works through search tools, like `ag`, `rg`, `ack`, `pt` and `grpe`."